OpenAI.Utilities/FunctionCalling/FunctionCallingHelper.cs

```diff
@@ -153,7 +153,7 @@ public static List<FunctionDefinition> GetFunctionDefinitions(Type type)
             throw new InvalidFunctionCallException($"Method '{functionCall.Name}' on type '{obj.GetType()}' not found");
         }
 
-        if (!methodInfo.ReturnType.IsAssignableTo(typeof(T)))
+        if (!typeof(T).IsAssignableFrom(methodInfo.ReturnType))
```

Comment:
IsAssignableTo not supported in .NetStandard 2.0 so reversed it to IsAssignableFrom for support.